heron/dsl/src/java/com/twitter/heron/dsl/impl/streamlets/ReMapStreamlet.java

```diff
@@ -0,0 +1,71 @@
+//  Copyright 2017 Twitter. All rights reserved.
```

Comment:
please rename it to RemapStreamlet.java and also the class.

Reply:
Done